src/Commands/Make/ClassCommand.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Nwidart\Modules\Commands\Make;
+
+use Illuminate\Support\Str;
+use Nwidart\Modules\Support\Config\GenerateConfigReader;
+use Nwidart\Modules\Support\Stub;
+use Nwidart\Modules\Traits\ModuleCommandTrait;
+
+class ClassCommand extends GeneratorCommand
+{
+    use ModuleCommandTrait;
+
+    /**
+     * The name and signature of the console command.
+     */
+    protected $signature = 'module:make-class
+        {--t|type=class : The type of class, e.g. class, service, repository, contract, etc.}
+        {--p|plain : Create the class without suffix (type)}
+        {--i|invokable : Generate a single method, invokable class}
+        {--f|force : Create the class even if the class already exists}
+        {name : The name of the class}
+        {module : The targeted module}';
+
+    /**
+     * The console command description.
+     */
+    protected $description = 'Create a new class';
+
+    protected $argumentName = "name";
+
+    public function getTemplateContents()
+    {
+        $module = $this->laravel['modules']->findOrFail($this->getModuleName());
+
+        return (new Stub('/app/Classes/Class.stub', [
```

Comment:
the Class.stub should be in the root, as the base path could be src

Reply:
The location of the stub file doesn't affect where the class will be placed. The final class file will be created in the path specified by the `generator.paths.app`.
I organize the stub file based on the default Laravel structure to help me and others get to the file easily later.